index.d.ts

```diff
@@ -3421,13 +3456,13 @@ declare namespace Eris {
     edit(options: Omit<EditChannelOptions, "icon" | "ownerID">, reason?: string): Promise<this>;
   }
 
-  export class TextChannel extends GuildChannel implements GuildTextable, Invitable {
+  export class TextChannel extends GuildChannel implements GuildTextable, Invitable, GuildPinnable {
     defaultAutoArchiveDuration: AutoArchiveDuration;
     lastMessageID: string;
     lastPinTimestamp: number | null;
     messages: Collection<Message<this>>;
     rateLimitPerUser: number;
-    topic: string | null;
+    topic?: string;
```

Comment:
```suggestion
    topic?: string | null;
```
topic is both optional and nullable
![image](https://user-images.githubusercontent.com/17226394/164949655-87f09e97-9631-409d-a66c-864e2f21d541.png)